docs/topics/native/native-debugging.md

```diff
@@ -44,89 +69,90 @@ Process 28473 stopped
-    (lldb) b -f hello.kt -l 1
-    Breakpoint 1: where = terminator.kexe`kfun:main(kotlin.Array<kotlin.String>) + 4 at hello.kt:2, address = 0x00000001000012e4
-    ```
+* By address:
 
-- by address
+  ```bash
+  (lldb) b -a 0x00000001000012e4
+  Breakpoint 2: address = 0x00000001000012e4
+  ```
 
-    ```bash
-    (lldb) b -a 0x00000001000012e4
-    Breakpoint 2: address = 0x00000001000012e4
-    ```
+* By regex. You might find it useful for debugging generated artifacts, like lambda (with the `#` symbol in the name):
 
-- by regex, you might find it useful for debugging generated artifacts, like lambda etc. (where used ``#`` symbol in name).
+  ```bash
+  (lldb) b -r main\(
+  3: regex = 'main\(', locations = 1
+    3.1: where = terminator.kexe`kfun:main(kotlin.Array<kotlin.String>) + 4 at hello.kt:2, address = terminator.kexe[0x00000001000012e4], unresolved, hit count = 0
+  ```
 
-    ```bash
-    3: regex = 'main\(', locations = 1
-      3.1: where = terminator.kexe`kfun:main(kotlin.Array<kotlin.String>) + 4 at hello.kt:2, address = terminator.kexe[0x00000001000012e4], unresolved, hit count = 0
-    ```
+### GDB
 
-### gdb
+* By regex:
 
-- by regex
+  ```bash
+  (gdb) rbreak main(
+  Breakpoint 1 at 0x1000109b4
+  struct ktype:kotlin.Unit &kfun:main(kotlin.Array<kotlin.String>);
+  ```
 
-    ```bash
-    (gdb) rbreak main(
-    Breakpoint 1 at 0x1000109b4
-    struct ktype:kotlin.Unit &kfun:main(kotlin.Array<kotlin.String>);
-    ```
+* By name is _unusable_ because `:` is a separator for the breakpoint by location:
 
-- by name __unusable__, because ``:`` is a separator for the breakpoint by location
-    
-    ```bash
-    (gdb) b kfun:main(kotlin.Array<kotlin.String>)
-    No source file named kfun.
-    Make breakpoint pending on future shared library load? (y or [n]) y
-    Breakpoint 1 (kfun:main(kotlin.Array<kotlin.String>)) pending
-    ```
+  ```bash
+  (gdb) b kfun:main(kotlin.Array<kotlin.String>)
+  No source file named kfun.
+  Make breakpoint pending on future shared library load? (y or [n]) y
+  Breakpoint 1 (kfun:main(kotlin.Array<kotlin.String>)) pending
+  ```
 
-- by location
+* By location:
 
-    ```bash
-    (gdb) b hello.kt:1
-    Breakpoint 2 at 0x100001704: file /Users/minamoto/ws/.git-trees/hello.kt, line 1.
-    ```
+  ```bash
+  (gdb) b hello.kt:1
+  Breakpoint 2 at 0x100001704: file /Users/minamoto/ws/.git-trees/hello.kt, line 1.
+  ```
 
-- by address
+* By address:
 
-    ```bash
-    (gdb) b *0x100001704
-    Note: breakpoint 2 also set at pc 0x100001704.
-    Breakpoint 3 at 0x100001704: file /Users/minamoto/ws/.git-trees/hello.kt, line 2.
-    ```
+  ```bash
+  (gdb) b *0x100001704
+  Note: breakpoint 2 also set at pc 0x100001704.
+  Breakpoint 3 at 0x100001704: file /Users/minamoto/ws/.git-trees/hello.kt, line 2.
+  ```
 
-## Stepping
+## Use stepping
 
-Stepping functions works mostly the same way as for C/C++ programs.
+Stepping through functions works mostly the same way as for C/C++ programs.
```

Comment:
Not sure if this is in scope of this PR but this seems a vague statement. Can we provide any more info on what is different if it's only "mostly the same"? 🤔